packages/cli/src/models/network/NetworkAppController.js

```diff
@@ -148,6 +164,28 @@ export default class NetworkAppController extends NetworkBaseController {
     log.error(`Possible migration method 'migrate' found in contract ${contractClass.contractName}. Remember running the migration after deploying it.`);
   }
 
+  _fetchOwnedProxies(packageName, contractAlias, proxyAddress) {
+    const proxies = this.networkFile.getProxies({ 
+      package: packageName || (contractAlias ? this.packageFile.name : undefined),
+      contract: contractAlias, 
+      address: proxyAddress
+    })
+
+    if (_.isEmpty(proxies)) {
+      log.info('No owned contract instances that match were found');
```

Comment:
Shouldn't this be `No matching contract instances were found`? 
We could also log what we were looking for to be clearer

Reply:
Note that you are repeating this validation in line 181, I think you can just delete this one since the `filter` above won't bother you in case no proxies were found at this point

Reply:
Validation in line 181 is about `ownedProxies`, not `proxies`